docs/interpreter/jdbc.md

```diff
@@ -723,6 +727,26 @@ Before Adding one of the below dependencies, check the Phoenix version first.
 </table>
 
 [Maven Repository: org.apache.tajo:tajo-jdbc](https://mvnrepository.com/artifact/org.apache.tajo/tajo-jdbc)
+## Object Interpolation
```

Comment:
I think you need an empty line before `##` or it will not be treated as header properly

Reply:
Added preceding blank line.